AnaProd/anaTupleProducer.py

```diff
@@ -74,6 +75,7 @@ def addAllVariables(dfw, syst_name, isData, trigger_class):
         dfw.DefineAndAppend( f"tau{leg_idx+1}_eta", f"static_cast<float>(httCand.leg_p4[{leg_idx}].Eta())")
         dfw.DefineAndAppend(f"tau{leg_idx+1}_phi", f"static_cast<float>(httCand.leg_p4[{leg_idx}].Phi())")
         dfw.DefineAndAppend(f"tau{leg_idx+1}_mass", f"static_cast<float>(httCand.leg_p4[{leg_idx}].M())")
+        dfw.DefineAndAppend(f"tau{leg_idx+1}_e", f"static_cast<float>(httCand.leg_p4[{leg_idx}].E())")
```

Comment:
do not store energy

Reply:
for the moment, until the mistery of b-jet discrepancies is totally fixed, I prefer to store at least bjet energy and at this point also taus one